base/version.jl

```diff
@@ -267,72 +267,3 @@ end
 
 libllvm_path() = ccall(:jl_get_libllvm, Any, ())
 
```

Comment:
The whitespace check is very particular about trailing newlines (there must be exactly one)
```suggestion
```